xbmc/interfaces/swig/AddonModuleXbmcgui.i

```diff
@@ -29,6 +29,7 @@
 #include "interfaces/legacy/Dialog.h"
 #include "interfaces/legacy/WindowXML.h"
 #include "input/Key.h"
+#include "input/ActionIDs.h"
```

Comment:
please move this one line up to match our [Code guidelines for includes](https://codedocs.xyz/xbmc/xbmc/code_guidelines.html#code_guidelines_1_3)